src/cascadia/TerminalCore/Terminal.hpp

```diff
@@ -182,13 +182,30 @@ class Microsoft::Terminal::Core::Terminal final :
     {
         Cell,
         Word,
-        Line
+        Line,
+        Viewport,
+        Buffer
     };
     void MultiClickSelection(const COORD viewportPos, SelectionExpansionMode expansionMode);
     void SetSelectionAnchor(const COORD position);
     void SetSelectionEnd(const COORD position, std::optional<SelectionExpansionMode> newExpansionMode = std::nullopt);
     void SetBlockSelection(const bool isEnabled) noexcept;
 
+    enum class Direction

```

Comment:
I _still_ think that we shouldn't have these two, and we should just be using the enums that are defined in `Microsoft.Terminal.Settings`. Since we can't currently have the `TerminalCore` project define it's own WinRT types, then we're kind of relying on putting them in the `TerminalSettings` project for now.

I'd LOVE the future there the TerminalCore project defines `ICoreSettings` and `Direction` (et. al) but we're just not there right now. 

(there's another thread about this higher in the PR, but github lost it apparently)